agent/go.mod

```diff
@@ -45,8 +45,8 @@ require (
 	golang.org/x/text v0.3.6 // indirect
 	golang.org/x/time v0.0.0-20170927054726-6dc17368e09b // indirect
 	golang.org/x/tools v0.1.5
-	google.golang.org/grpc v1.38.0 // indirect
```

Comment:
maybe we should also include go.sum modifications in this PR? i see some diffs from your previous PR https://github.com/aws/amazon-ecs-agent/pull/3406/files